base/models.py

```diff
@@ -41,13 +33,30 @@
 from wagtail.search import index
 from wagtailmedia.blocks import AbstractMediaChooserBlock
 
+from alerts.utils import get_alert
+from ask_a_librarian.utils import get_unit_chat_link
+from base.utils import get_hours_and_location
+from library_website.settings.base import (
+    HOURS_PAGE, LIBCAL_IID, PHONE_ERROR_MSG, PHONE_FORMAT,
+    POSTAL_CODE_ERROR_MSG, POSTAL_CODE_FORMAT, ROOT_UNIT
+)
+from units.utils import get_default_unit
+
 # Helper functions and constants
 BUTTON_CHOICES = (
     ('btn-primary', 'Primary'),
     ('btn-default', 'Secondary'),
     ('btn-reserve', 'Reservation'),
 )
 
+NEWS_CHOICES = (
+    ('', '--------'),
+    ('library_kiosk', 'Library'),
+    ('law_kiosk', 'Law'),
+    ('crerar_kiosk', 'Crerar'),
```

Comment:
EL requested this label be changed to "Sciences"

Reply:
Right, so the way it will work is that on the Eckhart page they will choose the Crerar option from the dropdown. Previously it was a url field linking to Crear's feed. So, the real difference here is that it will be a dropdown in the Wagtail admin, instead of a UrlField. But, they'll still be able to use those options on their pages. So, Eckhart can have the Crerar feed and SSA can have the main Library feed.

Reply:
Great! I see this drop-down now in "News feed source" in the widgets tab. I dig this solution.

Reply:
Ah yes, of course. I didn't change the link in the global navbar. Good catch! I'll change that and the Crerar label.